tt_test.go

```diff
@@ -0,0 +1,672 @@
+package tupleconv_test
+
+import (
+	"github.com/google/uuid"
+	dec "github.com/shopspring/decimal"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/go-tarantool/datetime"
+	"github.com/tarantool/go-tarantool/decimal"
+	"github.com/tarantool/go-tupleconv"
+	"math/big"
+	"testing"
+	"time"
+)
+
+type ttTestCase struct {
+	value      string
+	expected   any
+	isNullable bool
+	isErr      bool
+}
+
+func helperTestStringsToTTConvFactory(
```

Comment:
It seems to me it is too complicated for tests.

A map of testcases, 3 levels of tests... It should be easier.

Reply:
Tests were reworked.